AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -232,6 +232,10 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         stateMachine.notify(action: .finished)
     }
 
+    func isSyncing() -> Bool {
+        stateMachine.state.displayName != State.notStarted.displayName
```

Comment:
Can we directly check the enum cases instead of checking the property `displayName`?


```
    func isSyncing() -> Bool {
        if case .notStarted = stateMachine.state {
            return false
        }
        return true
    }
```

Reply:
will change to that.